src/main/steps/tasklistresponse/utils.ts

```diff
@@ -40,24 +39,26 @@ export const mapDataInSession = (userCase: CaseWithId, userId: UserDetails['id']
     Object.assign(userCase, mapContactPreference(partyDetails));
   }
 };
-
-const setDataInSession = (userCase: CaseWithId, partyDetails: PartyDetails): void => {
-  if (partyDetails?.response?.safetyConcerns) {
-    Object.assign(userCase, mapSafetyConcernsDetails(partyDetails));
+function setDataInSession(userCase: CaseWithId, partyDetails: PartyDetails) {
+  if (partyDetails?.response) {
+    //Object.assign(userCase, { ...mapSafetyConcernsDetails(partyDetails), ...userCase });
+    if (partyDetails?.response.respondingCitizenAoH) {
```

Comment:
u can use _ get method to check and keep the conditional check simple

Reply:
addressed